subsys/net/lib/aws_iot/Kconfig

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig AWS_IOT
+	bool "AWS IoT library"
+	select CLOUD_API
+	select MQTT_LIB
+	select MQTT_LIB_TLS
+	select AT_CMD
```

Comment:
I think we should try to avoid this dependency to `AT_CMD`

Reply:
In that case, is there any alternative way of getting the device IMEI for the shadow topics?

Reply:
I think this should perhaps go into a new field in `struct cloud_backend_config`. Alternatively, the `user_data` pointer can be used for whater one wishes, for example a null-terminated string containing user ID. A bit hacky, perhaps, but doable.